x-pack/qa/integration/support/helpers.rb

```diff
@@ -56,8 +57,13 @@ def elasticsearch(options = {})
 end
 
 def start_es_xpack_trial
+  puts "Starting trial"
```

Comment:
@andrewvc I guess we should remove the `puts` again to keep the noise from the build down? Other than that , LGTM :)